packages/teams-js/src/public/meeting.ts

```diff
@@ -99,7 +99,25 @@ export namespace meeting {
    * Hide from docs
    * Data structure to represent call details
    */
-  export type ICallDetails = IMeetingOrCallDetailsBase<CallType>;
+  export interface ICallDetails extends IMeetingOrCallDetailsBase<CallType> {
+    /**
+     * @hidden
+     * Unique identifier for the original caller
```

Comment:
Do you mean that we should update the description to elaborate on the format of the values returned (i.e. string, but in UUID format), or are you asking if we can define these values as more specific data types (instead of just "string")?

I will definitely do the former, but for the latter, we unfortunately don't have more specific data types defined for those fields, even internally within Teams.